app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_incoming: list) -> float:
+        income = 0.0
+        for car in cars_incoming:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+                print(self.calculate_washing_price(car))
+        return round(income, 1)
+
+    def calculate_washing_price(self, car_calculate: Car) -> float:
```

Comment:
we have a better visualization for a long string in the documentation for this task.
Good example:
how_much_fuel_needed = (max_fuel_consumption
                        * height_fuel_consumption_coeficient
                        * distance_to_the_destinatoin
                        / (plan_max_speed
                           - wind_awerage_speed
                           * wind_angle_coefisient)
                        * overlap_coeficient)